### PR TITLE
Converted to es6 classes/modules and added imports

### PR DESCRIPTION
**Pull request by eleanor-heath**
Part of https://github.com/adaptlearning/adapt_framework/issues/2808

### CODE REVIEW

File: js/ResourcesView.js [hunk 1]
```diff
@@ -0,0 +1,51 @@
+import Adapt from 'core/js/adapt';
+export default class ResourcesView extends Backbone.View {
```
Comment (oliverfoster) [inline js/ResourcesView.js:2]:
```suggestion
import Adapt from 'core/js/adapt';

export default class ResourcesView extends Backbone.View {
```

Add line to differentiate between import and export statements.

File: js/ResourcesHelpers.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
+import Handlebars from 'handlebars';
+import Adapt from 'core/js/adapt';
+
+class ResourcesHelpers {
```
Comment (oliverfoster) [inline js/ResourcesHelpers.js:4]: This should be left as an `Object` as it never needs to be instantiated, it's just acting as "hash" / a named list for a loop.

File: js/adapt-contrib-resources.js [hunk 3]
```diff
@@ -1,31 +1,15 @@
...
-        collection: resourcesCollection
-      }).$el);
-    });
-
-  }
-
-  function initResources() {
-
-    var courseResources = Adapt.course.get('_resources');
+class Resources {
```
Comment (oliverfoster) [inline js/adapt-contrib-resources.js:5]: What we've tended to do with these kind of controller classes is make them into a singleton.

https://github.com/adaptlearning/adapt_framework/blob/master/src/core/js/headings.js#L17

The class inherits the ability to be an event listener from `Backbone.Controller` and its `initialize` function is executed when the `Backbone.Controller` is instantiated. At the end of the file the singleton is instantiated and returned.

**Review by oliverfoster — CHANGES_REQUESTED** [review]

File: js/adapt-contrib-resources.js [hunk 4]
```diff
@@ -2,7 +2,12 @@ import Adapt from 'core/js/adapt';
 import ResourcesView from './ResourcesView';
 import ResourcesHelpers from './ResourcesHelpers';
 
-class Resources {
+class Resources extends Backbone.Controller {
+  
+  initialize() {
+    this.listenTo(Adapt, 'adapt:start', this.initResources);
```
Comment (oliverfoster) [inline js/adapt-contrib-resources.js:8]: Could you also change 
` Adapt.on('resources:showResources', () => {`
to:
`this.listenTo(Adapt, 'resources:showResources', () => {`

File: js/ResourcesHelpers.js [hunk 5]
```diff
@@ -1,43 +1,44 @@
...
-  /**
-   * IE doesn't support the 'download' attribute
-   * https://github.com/adaptlearning/adapt_framework/issues/1559
-   * and iOS just opens links with that attribute in the same window
-   * https://github.com/adaptlearning/adapt_framework/issues/1852
-   */
-  resources_force_download(resource, block) {
-    if (Adapt.device.browser === 'internet explorer' || Adapt.device.OS === 'ios') {
-      return block.inverse(this);
+define([
```
Comment (oliverfoster) [inline js/ResourcesHelpers.js]: These should still be imports.

File: js/ResourcesHelpers.js [hunk 6]
```diff
@@ -1,43 +1,44 @@
...
+  for (const name in helpers) {
+    Handlebars.registerHelper(name, helpers[name]);
+  }
 
-helpers.filter(name => name != 'constructor').forEach(value => {
-  Handlebars.registerHelper(value, new ResourcesHelpers()[value]);
+  return helpers;
 });
-
-export default ResourcesHelpers;
```
Comment (oliverfoster) [inline js/ResourcesHelpers.js]: `export default helpers;` should suffice here

**Review by oliverfoster — CHANGES_REQUESTED** [review]

**Review by oliverfoster — APPROVED** [review]

**Comment by oliverfoster** [comment]
Oh, can you put the package.json back please?

**Review by oliverfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

**Review by ethan-lp — APPROVED** [review]
👀

**Pull request merged**